mysql-test/suite/galera/t/galera_as_master_gtid_change_master.test

```diff
@@ -54,3 +54,14 @@ DROP TABLE t1;
 
 STOP SLAVE;
 RESET SLAVE ALL;
+RESET MASTER;
+
+--connection node_2
+SET GLOBAL wsrep_on=OFF;
+RESET MASTER;
+SET GLOBAL wsrep_on=ON;
+
+--connection node_1
+SET GLOBAL wsrep_on=OFF;
+RESET MASTER;
+SET GLOBAL wsrep_on=ON;
```

Comment:
Changes not related to 10715, please do this in other commit , (same for all unrelated test case changes )

Reply:
Is this needed for clean setup for next test ?

Reply:
This test was disabled and not maintained. This is cleanup for next test